### ISSUE TITLE
Import as gradle plugin

### ISSUE THREAD

**Issue by dantp-ai** [spiritinlife/EasySlideShow4Android#1]
It would be nice if the library could be imported in Android Studio as a Gradle plugin

**Comment by spiritinlife** [comment]
Yes that would be better. But i am waiting until i find time and complete some todos or someone else does. If this is going to become a library it needs to be more generic.
